reloadRecipe.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+echo "Run this script from the same directory as your recipe metadata.json"
+echo "Provide an environment shortname which is at least Flow V5.11."
+echo "The first parameter must either be the environment shortname or an integer greater than 0 (the number of days to regard as 'recent' when fetching recent recipeExecutions, defaults to 1 day if not supplied)"
+# Takes in up to three parameters: 
+#   - environment ENV, defaults to 'local'
+#   - number of days to fetch recipeExecutions for RECENT, defaults to 1
+#   - a username to use to filter recipeExecutions RECIPEUSER
+# Use the following flags to include each parameter:
+#  ENV: -e
+#  RECENT: -d
+#  RECIPEUSER: -u
+ENV='local'
+RECIPEUSER='none'
+
+while getopts d:u:e: flag
+do
+  case "${flag}" in
+    d) RECENT=${OPTARG};;
+    u) RECIPEUSER=${OPTARG};;
+    e) ENV=${OPTARG};;
+  esac
+done
+
+if [ -z "$RECENT" ]; then RECENT=1; fi
+if ! [[ $RECENT =~ ^[0-9]+$ && $RECENT -gt 0 ]]; then echo "Days parameter, if provided, must be an integer greater than 0"; exit; fi
+
+RECIPE_FAMILY=$(basename "`pwd`")
+RECIPE=$(jq -r ".id" metadata.json)
+echo "Refreshing ${RECIPE_FAMILY} - ${RECIPE}"
+pushd ..
```

Comment:
It's common to add ` > /dev/null` after `pushd` and `popd` commands to avoid adding their output to the script's output.  If either fail, they will print out errors to help in diagnosing the problem.

Reply:
Added, thanks @timjstewart1024